config/routes.rb

```diff
@@ -0,0 +1,16 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  root 'homepage#index'
+  resources :works, :users, :votes
```

Comment:
Actually, when you declare `resources :works` and `resources :users` below, it accomplishes the same as the line above, even if you _are_ describing a nesting, so the line above is redundant!